README.md

```diff
@@ -108,16 +114,18 @@ If you are going to use this tool frequently you may want to have command auto-c
 
 * Cygwin:
 
-  Make sure the `bash-completion` package is installed (Cygwin doesn't installed it by default) and type:
-  ```sh
-  $ vt completion bash > /usr/share/bash-completion/completions/vt
-  ```
+Make sure the `bash-completion` package is installed (Cygwin doesn't installed it by default) and type:
+
+```sh
+$ vt completion bash > /usr/share/bash-completion/completions/vt
+```
```

Comment:
This change breaks the formatting of our readme. Revert it.

Reply:
Nice catch, fixed it. Made some minor modifications too, feel free to let me know if I need to revert anything.